### ISSUE TITLE
ImGuiInputTextFlags conflict with InputTextMultiline

### ISSUE THREAD

**Issue by kfsone** [ocornut/imgui#4409]
```
   static char b1[256], b2[256], b3[256];
   ImGui::InputTextMultiline("##t1", b1, 255, ImVec2(0,0), ImGuiInputTextFlags_None);
   ImGui::InputTextMultiline("##t2", b2, 255, ImVec2(0,0), ImGuiInputTextFlags_CharsHexadecimal);
   ImGui::InputTextMultiline("##t3", b3, 255, ImVec2(0,0), ImGuiInputTextFlags_CharsHexadecimal | ImGuiInputTextFlags_CtrlEnterForNewline);
```

In the first, you can enter multi-line text as expected.

In the second, you can only enter hexadecimal characters -- not including enter.

In the third, you still can't insert a newline.

Suggestion: If these flags are set, allow those characters to skip the named-filter checks.

PR to follow.

**Comment by kfsone** [comment]
see https://www.github.com/kfsone/imgui-stuff for a simple demo of the problem.

**Comment by ocornut** [comment]
Thank you Olivier,
This is now merged as 7e9e1ff1c2
